docs/user-guide/time-and-costs/my-time-tracking/README.md

```diff
@@ -53,6 +53,10 @@ At the bottom of each day a sum of hours per day is displayed. If week or month
 
 ![Sums of hours displayed at the bottom of the Calendar tab under My time tracking module in OpenProject](openproject_user_guide_my_time_tracking_module_calendar_tab_total_sums.png)
 
+If you are currently tracking time for a specific work package, you will also see the ongoing time tracking timer for the respective work package. The work package in question also has a different background color for easier differentiation. 
```

Comment:
I would phrase the `If you are currently tracking time for a specific work package` a bit different to make clear we are talking here about a running log time timer running. I would say something like `If you are currently tracking time with a timer for a specific work package`.